src/main/java/ch/bfh/bti7081/s2018/green/presenters/JournalPresenter.java

```diff
@@ -47,7 +45,8 @@ private void addEntry() {
         }
     }
     private void enteredView() {
-	    JournalEntryManager manager = new JournalEntryManager();
-        view.setJournalEntries(manager.findBy(data.getCurrentPatient()));
+        // will be called when corresponding view is about to open
+        // use this method to populate the view-elements with data
+        //view.setJournalEntries(data.getCurrentPatient().getJournalEntries());
```

Comment:
Why did you comment this out?

Reply:
There was some difference between my version of master and master here.
Fixed by apply most recent version of master.